app_server/app/adapters/socket/Room.js

```diff
@@ -75,4 +75,21 @@ Class.removeAllClients = function() {
   this.__clients = {};
 };
 
+Class.getNumberOfUsers = function() {
+  let users = {};
+  let count = 0;
+  for (var socketId in this.getClients()) {
```

Comment:
Should this be `var` instead of `let` or `const`?
Seeing as the rest are in ES6.